lib/expansion_rules.rb

```diff
@@ -59,25 +59,25 @@ def recurring
     { document_type: :world_location,             fields: [:content_id, :title, :schema_name, :locale, :analytics_identifier] },
   ].freeze
 
-  def root_reverse_links
+  def reverse_links
     REVERSE_LINKS.values
   end
 
   def reverse_link_type(link_type)
     REVERSE_LINKS[link_type.to_sym]
   end
 
-  def un_reverse_link_type(link_type)
+  def unreverse_link_type(link_type)
```

Comment:
That explanation is really helpful, I think I'm struggling with the word `unreverse`.
Given that the purpose of the method is to return the opposing type of link `opposing_link_type` might be enough, that would map `children` -> `parent` and vice versa, do we really care that it's relevant to to reverse links? Like I said, not important for this PR.